packages/eas-cli/src/commands/account/login.ts

```diff
@@ -25,7 +29,46 @@ export default class AccountLogin extends EasCommand {
       flags: { sso },
     } = await this.parse(AccountLogin);
 
-    const { sessionManager } = await this.getContextAsync(AccountLogin, { nonInteractive: false });
+    const {
+      sessionManager,
+      maybeLoggedIn: { actor },
+    } = await this.getContextAsync(AccountLogin, { nonInteractive: false });
+
+    if (sessionManager.getAccessToken()) {
+      Log.warn(
```

Comment:
IMO we should just throw an error when `eas login` is run when the `EXPO_TOKEN` is set.